Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/UserPreferences.java

```diff
@@ -142,6 +239,129 @@ public static void SavePrefsToIni(Context ctx)
 		// Current CPU core being used. Falls back to interpreter upon error.
 		String currentEmuCore = prefs.getString("cpuCorePref", "0");
 
+		// Current GC modifier range setup. Falls back to 50.000000 upon error.
+		String MainRange_0 = prefs.getString("MainRange_0", "50.000000");
```

Comment:
If you look at the xml file, all the naming is done as ButtonName_Controller#, eg:
MainLeft_0 , MainRight_0, MainLeft_1, MainLeft_1, etc.
I see no reason not to stick to the naming style originally chosen, so that the naming style thorughout the xml file stays consistent.